### PR TITLE
[Ready for Review] Added Redis cache samples

### PR DESCRIPTION
**Pull request by hovsepm**

### CODE REVIEW

File: azure-samples/src/main/java/com/microsoft/azure/management/rediscache/samples/ManageRedisCache.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    /**
+     * Main entry point.
+     * @param args the parameters
+     */
+    public static void main(String[] args) {
+
+        final String redisCacheName1 = Utils.createRandomName("rc1");
+        final String redisCacheName2 = Utils.createRandomName("rc2");
+        final String redisCacheName3 = Utils.createRandomName("rc3");
+        final String rgName = Utils.createRandomName("rgRCMS");
```
Comment (selvasingh) [inline azure-samples/src/main/java/com/microsoft/azure/management/rediscache/samples/ManageRedisCache.java]: Suggest rgRCMC as the resource group prefix

File: azure-samples/src/main/java/com/microsoft/azure/management/rediscache/samples/ManageRedisCache.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                        .withRegion(Region.US_CENTRAL)
+                        .withNewResourceGroup(rgName)
+                        .withPremiumSku(2)
+                        .create();
+
+                System.out.println("Created a Redis Cache:");
+                Utils.print(redisCache3);
+                // ============================================================
+                // List Redis Caches
+
```
Comment (selvasingh) [inline azure-samples/src/main/java/com/microsoft/azure/management/rediscache/samples/ManageRedisCache.java]: Suggest that you update this comment with what is happening with Premium SKU inside the list walk

File: azure-samples/src/main/java/com/microsoft/azure/management/rediscache/samples/ManageRedisCache.java (hunk 1 above)
Comment (hovsepm) [inline azure-samples/src/main/java/com/microsoft/azure/management/rediscache/samples/ManageRedisCache.java]: fixed

File: azure-samples/src/main/java/com/microsoft/azure/management/rediscache/samples/ManageRedisCache.java (hunk 2 above)
Comment (hovsepm) [inline azure-samples/src/main/java/com/microsoft/azure/management/rediscache/samples/ManageRedisCache.java]: fixed.

**Pull request merged**